lang/lang.go

```diff
@@ -159,6 +166,21 @@ func AddTranslationsFS(fs embed.FS, dir string) (retErr error) {
 	return retErr
 }
 
+// SetLanguageOrder allows an app to set the order in which translations are checked in case no locale matches.
+// Since 2.6
+func SetLanguageOrder(order []string) {
+	languageOrder = order
+	updateLocalizer()
+}
+
+// SetPreferredLocale allows an app to set the preferred locale for translations, overwriting the System Locale.
+// locale can be in format en_US_someVariant, en_US, en-US-someVariant, en-US, en
+// Since 2.6
```

Comment:
This isn't the right format for a since line, missing : and the blank comment line above - see https://github.com/fyne-io/fyne/wiki/Contributing (or other Since: lines in the codebase).